app/views/play/level/modal/CourseVictoryModal.coffee

```diff
@@ -98,6 +101,16 @@ module.exports = class CourseVictoryModal extends ModalView
         rewardsView.on 'continue', @onViewContinue, @
         @views.push(rewardsView)
 
+    # get next level for voyager course, no `nextAssessment` in voyager
+    if utils.voyagerCourseIDs.includes(@courseID) and @classroom
+      currentLevel = @classroom.get('courses')?.find((c) => c._id == @courseID)?.levels?.find((l) => l.original == @level.get('original'))
+      nextLevelOriginal = voyagerUtils.getNextLevelOriginalForLevel(currentLevel)
```

Comment:
Similar thoughts about one vs many next levels.

Reply:
Not yet, in the future if we have multiple next levels, the logic will depend on how do we want the game flow to be.
But I would want to handle all that complicated logic in `getNextLevelOriginalForLevel` as much as possible, and just return whats relevant to course victory modal.